src/main/kotlin/com/jetbrains/micropython/devices/Esp32DeviceProvider.kt

```diff
@@ -0,0 +1,51 @@
+package com.jetbrains.micropython.devices
+
+import com.intellij.execution.configurations.CommandLineState
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.OSProcessHandler
+import com.intellij.execution.runners.ExecutionEnvironment
+import com.intellij.openapi.projectRoots.Sdk
+import com.jetbrains.micropython.run.MicroPythonRunConfiguration
+import com.jetbrains.micropython.run.getMicroUploadCommand
+import com.jetbrains.micropython.settings.MicroPythonTypeHints
+import com.jetbrains.micropython.settings.MicroPythonUsbId
+import com.jetbrains.python.packaging.PyPackageManager
+import com.jetbrains.python.packaging.PyRequirement
+
+/**
+ * @author Gor-Ren
+ */
+class Esp32DeviceProvider : MicroPythonDeviceProvider {
+  override val persistentName: String
+    get() = "ESP32"
+
+  override val documentationURL: String
+    get() = "https://github.com/vlasovskikh/intellij-micropython/wiki/ESP32"  // FIXME: add wiki page
+
+  override val usbIds: List<MicroPythonUsbId>
+    get() = listOf(MicroPythonUsbId(0x1A86, 0x7523),
+                   MicroPythonUsbId(0x10C4, 0xEA60),
+                   MicroPythonUsbId(0x0403, 0x6001))
```

Comment:
These are copied from the ESP8266. 

To find the values for an ESP32, I connected MCUs by USB and ran `$ cat /sys/bus/usb-serial/devices/ttyUSB0/../uevent`. I tested for both an ESP8266 and an ESP32 and got an output indicating `PRODUCT=10c4/ea60/100`. 

This corresponds to the second element in this list. Auto-detect of my ESP32 subsequently worked. Does that mean these copied values can be kept the same? I am a little surprised at that.